Orange/widgets/model/owsvm.py

```diff
@@ -29,6 +31,10 @@ class OWSVM(OWBaseLearner):
     class Outputs(OWBaseLearner.Outputs):
         support_vectors = Output("Support vectors", Table, explicit=True)
 
+    class Warning(OWWidget.Warning):
+        sparse_data = Msg('Input data is sparse, default preprocessing is to scale it.')
+        outdated_learner = Msg("Press Apply to submit changes.")
```

Comment:
Is `outdated_learner` used anywhere?

Reply:
It is used in the parent class. I redefine it here, since i needed to add sparse data warning.

Reply:
Inherit from OWBaseLearner.Warning in the same way as is done for Outputs.